### PR TITLE
[Release] Fix cluster.yaml 

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

Fix cluster.yaml of stress_test/long_running_test to be running with releaser. 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/long_running_tests/run.sh [hunk 1]
```diff
@@ -62,5 +62,6 @@ pip install --upgrade pip
 pip install -U tensorflow==1.14
 pip install -q -U "$wheel" Click
 pip install -q "ray[all]" "gym[atari]"
-python "workloads/$workload.py"
+cd ..
```
Comment (simon-mo) [inline release/long_running_tests/run.sh:65]: which directory are we supposed to be in? why the cd ..?

File: release/stress_tests/cluster.yaml [hunk 2]
```diff
@@ -44,7 +44,7 @@ auth:
 # http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.ServiceResource.create_instances
 head_node:
     InstanceType: m4.16xlarge
-    ImageId: ami-06d51e91cea0dac8d # Ubuntu 18.04
+    ImageId: ami-042d0d6196f494652 # Custom ami
```
Comment (simon-mo) [inline release/stress_tests/cluster.yaml:47]: can't we just let ray or the provider auto-fill?

**Review by simon-mo — APPROVED** [review]

File: release/long_running_tests/run.sh (hunk 1 above)
Comment (rkooo567) [reply release/long_running_tests/run.sh:65]: There is a line that calls `cd wrk`, so this fails to find the workloads/*.py file.

File: release/stress_tests/cluster.yaml (hunk 2 above)
Comment (rkooo567) [reply release/stress_tests/cluster.yaml:47]: If we remove this line, is it auto-filled?

Comment (simon-mo) [reply release/stress_tests/cluster.yaml:47]: Yes

**Pull request merged**